gym/envs/toy_text/blackjack.py

```diff
@@ -90,15 +95,25 @@ def seed(self, seed=None):
 
     def step(self, action):
         assert self.action_space.contains(action)
-        if action:  # hit: add a card to players hand and return
+        if self.double_down:
```

Comment:
minor nitpick - given the `assert self.action_space.contains(action)` above, if action == 2, then self.double_down should always be true, so the outer if statement is unnecessary here

Reply:
Thanks, fixed in new commit